pkg/logs/input/util.go

```diff
@@ -0,0 +1,33 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
nit: maybe rename the file `service_name.go`, just to avoid the generic `util` name?